test/ExternalLink.spec.tsx

```diff
@@ -4,7 +4,7 @@ import '@testing-library/jest-dom/extend-expect';
 import ExternalLink from '../src/ExternalLink';
 
 describe('<ExternalLink />', () => {
-  const createExternalLink = (href: string = 'href', children?: ReactNode): ChildNode => {
```

Comment:
No need to remove this. Lets keep it as it was.

Reply:
I think npx eslint --fix removed it